packages/react-widgets/src/models/FormulaModel.js

```diff
@@ -16,7 +12,7 @@ export const getFormula = async (props) => {
     filters,
     opts,
     viewportFilter,
-    viewportFeatures,
+    dataSource,
```

Comment:
Same as in previous. I think we should probably preserve both behaviours, binary with workers and current functions for plain common features. The strategy for the calculation method could be hidden, depending on the params. What do you think?